search/templates/cosmos/header.html

```diff
@@ -113,14 +120,27 @@
         <script type="text/javascript"
                 src="//platform-api.sharethis.com/js/sharethis.js#property=5a8129cbe45fbb001342fed3&product=custom-share-buttons"></script>
         <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
-        <script>
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js" type="text/javascript"></script>
```

Comment:
Two versions of jQuery are being loaded. No conflict must be enabled. It will be nice if we use one version :+1: